packages/core/src/SlideBox/SlideBox.tsx

```diff
@@ -24,6 +24,9 @@ export type SlideBoxProps = {
   arrowPosition?: ArrowPosition
   slideScrollNum?: number
   mobileSlideScrollNum?: number
+  overflowAllowanceX?: number
+  overflowAllowanceY?: number
+  overflowAllowanceTop?: number
```

Comment:
n-b would including `px` in these prop names clarify the type of value expected?

Reply:
Updated prop names to include "px"